src/wagtail_analytics/migrations/0005_analyticssettings_plausible_api_key.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.1.3 on 2023-03-13 10:47
```

Comment:
there are a lot of new migrations, only 1 is needed. so migrate back, remove the old ones and make 1 valid new one in the PR